app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+ 
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: 'login' }) do |f| %>
+  <div class="field login">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Enter your email" %>
+  </div>
+
+  <div class="field login">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Enter your password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field rememberbox">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions login-btn">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<div class= "links-bottom">
```

Comment:
- [ ] As requested by the preivous reviewer I think it would be great if you update your styles here since it is not styled well so kindly add some styles to make your project human kind ✅🔥🚀


![image](https://user-images.githubusercontent.com/58553711/217976954-833a95ed-28ce-4502-b74d-74690f58e5c7.png)